previous/__main__.py

```diff
@@ -0,0 +1,37 @@
+from os import getenv, listdir
+from os.path import isfile
+
+from aiohttp import ClientSession
+
+from nextcord import Intents
+from nextcord.ext.commands import Bot
+
+
+class MyBot(Bot):
```

Comment:
consider naming this something more descriptive?
such as `Previous`

Reply:
@ooliver1 ?

Reply:
i got a notification, i did not need a ping again